connectivity/netsocket/source/TLSSocketWrapper.cpp

```diff
@@ -427,7 +433,7 @@ void TLSSocketWrapper::print_mbedtls_error(MBED_UNUSED const char *name, MBED_UN
 {
 // Avoid pulling in mbedtls_strerror when trace is not enabled
 #if defined FEA_TRACE_SUPPORT && defined MBEDTLS_ERROR_C
-    char *buf = new char[128];
+    char buf[128];
```

Comment:
you're allocating on the stack, that's quite a buffer and besides look at line 439